docs/operators/kafkaInputOperator.md

```diff
@@ -398,33 +398,39 @@ public class ExampleKafkaApplication implements StreamingApplication
   }
 }
 ```
-Below is the configuration for “test” Kafka topic name and
-“localhost:9092” is the Broker:
+Below is the configuration for topic “test” and broker “localhost:9092”:
 
 ```xml
 <property>
-  <name>dt.operator.MessageReader.prop.topics</name>
+  <name>apex.operator.MessageReader.prop.topics</name>
   <value>test</value>
 </property>
 
 <property>
-  <name>dt.operator.KafkaInputOperator.prop.clusters</nam>
+  <name>apex.operator.KafkaInputOperator.prop.clusters</name>
   <value>localhost:9092</value>
 </property>
 ```
 
-By adding following lines to properties file, Kafka Input Operator supports multi-topic and multi-cluster:
+Following lines are an example for multi-topic and multi-cluster:
```

Comment:
It would be a smoother read if we replace this line with something like this:
"Multiple topics can be specified as a comma-separated list; similarly, multiple clusters can be specified as a semicolon-separated list; for example:"